src/JSInterop/Microsoft.JSInterop/src/IDotNetObjectRef.cs

```diff
@@ -7,6 +7,5 @@ namespace Microsoft.JSInterop
 {
     internal interface IDotNetObjectRef : IDisposable
```

Comment:
Is this still used?

Reply:
We use the marker interface in the dispatcher: https://github.com/aspnet/Extensions/blob/master/src/JSInterop/Microsoft.JSInterop/src/DotNetDispatcher.cs#L243. We could have it do the same check as the one in the factory

Reply:
That's fine, I just wanted to make sure it wasn't a mistake.